students/eunhyekim/users/views.py

```diff
@@ -11,14 +11,21 @@
 class SignInView(View):
     def post(self, request):
         try:
-            data = json.loads(request.body)
-            
-            if not User.objects.filter(email = data["email"], password = data["password"]).exists():
-                return JsonResponse({'message':'INVALID_USER'}, status = 401)
-            return JsonResponse({'message':'SUCCESS'},status = 200)
+            data     = json.loads(request.body)
+            password = User.objects.get(email = data["email"]).password 
+            user_id  = User.objects.get(email = data["email"]).id 
+        
+            if bcrypt.checkpw(data['password'].encode('utf-8'), password.encode('utf-8')) == True:
+                access_token = jwt.encode({'id':user_id}, 'secret', algorithm = 'HS256')
+          
```

Comment:
1. 위에서 드린 피드백데로 아래 코드도 수정해주세요~ (password 부분)
2. `if bcrypt.checkpw(data['password'].encode('utf-8'), password.encode('utf-8')) == True` 보다는 아래와 같이 `==True`를 빼주세요. 동일하게 동작합니다!

```suggestion
            if bcrypt.checkpw(data['password'].encode('utf-8'), password.encode('utf-8')):
                access_token = jwt.encode({'id':user_id}, 'secret', algorithm = 'HS256')
          
```

Reply:
네! 수정하였습니다!